src/webview/WebViewScreen.tsx

```diff
@@ -145,6 +151,37 @@ function WebViewScreen({ route, navigation }: Props) {
     webViewRef.current?.goBack()
   }
 
+  const openActionSheet = () => {
+    Platform.OS === 'ios' ? OpenActionSheetiOS() : toggleBottomSheet()
+    ValoraAnalytics.track(WebViewEvents.webview_more_options, { currentUrl: currentUrl })
+  }
+
+  // iOS Action sheet
+  const OpenActionSheetiOS = () => {
```

Comment:
Fixed in https://github.com/valora-inc/wallet/pull/2565/commits/8a36cf8a3596e13c40534c2b37bf18ad499d2f9d! Opted for `openActionSheetiOS` instead of `openActionSheetIos` because 'Ios' looks strange to me.